packages/core/useMouse/index.ts

```diff
@@ -52,7 +52,7 @@ export interface UseMouseOptions extends ConfigurableWindow, ConfigurableEventFi
 }
 
 const UseMouseBuiltinExtractors: Record<UseMouseCoordType, UseMouseEventExtractor> = {
-  page: event => [event.pageX, event.pageY],
+  page: event => [event.clientX, event.clientY],
```

Comment:
But shouldn't the users change the extract to `client` instead?

Reply:
Sorry for this, i will revert this commit shortly.